### ISSUE TITLE
Make the map edge to edge

### ISSUE THREAD

**Issue by teolemon** [aleene/FoodViewer#403]
![image](https://cloud.githubusercontent.com/assets/1689815/24072841/7a593ae2-0bed-11e7-9535-3a8e9ab906b3.png)

**Comment by aleene** [comment]
Yeah, I have a problem with detecting the right widths at the right moment.

**Comment by aleene** [comment]
Map is suppressed for the moment. Gives a memory leak. Is an Apple issue. Will try with a new version of Xcode
